repl/src/main/scala/ai/h2o/sparkling/repl/PatchUtils.scala

```diff
@@ -56,7 +56,11 @@ private[repl] object PatchUtils {
     val patcher: Patcher = (obj: AnyRef, clz: Class[_]) => {
       val f = clz.getDeclaredField("REPLClass")
       f.setAccessible(true)
-      f.set(obj, OUTER_SCOPE_REPL_REGEX)
+      try {
+        f.set(obj, OUTER_SCOPE_REPL_REGEX)
+      } catch {
+        case _: Throwable => // we have already patched once
```

Comment:
Shouldn't we be more specific with the exception type?

Reply:
Yup, will improve